libs/libcommon/src/libcommon/utils.py

```diff
@@ -100,6 +100,9 @@ class PaginatedResponse(TypedDict):
     num_rows_per_page: int
 
 
+MAX_NUM_ROWS_PER_PAGE = 100
```

Comment:
should we handle this with `FIRST_ROWS_MAX_NUMBER`? Or hardcode `FIRST_ROWS_MAX_NUMBER` and use `MAX_NUM_ROWS_PER_PAGE` instead (I think so)

Reply:
hmmm: but we set `FIRST_ROWS_MAX_NUMBER` to 4 in the tests...

Reply:
yes, sure!